CHANGELOG.md

```diff
@@ -16,6 +16,15 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.3.5] - 2021-01-13
```

Comment:
Technically this adds new functionality, so per semver this should increase the minor version number.

```suggestion
## [2.4.0] - 2021-01-22
```